sdk/keyvault/keyvault-keys/README.md

```diff
@@ -171,65 +167,102 @@ const result = await client.createKey(keyName, "RSA");
 console.log("result: ", result);
 ```
 
-### Get a key
+The second parameter sent to `createKey` is the type of the key. Keys can
+either be of type: `EC`, `EC-HSM`, `RSA`, `RSA-HSM` or `oct`.
```

Comment:
The first 4 look right, but not sure what the 'oct' is (and it's not listed in https://docs.microsoft.com/en-us/azure/key-vault/about-keys-secrets-and-certificates)

Reply:
Oct is part of: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/src/core/models/index.ts#L2266 but I'll remove it!